ai/playbooks/environment-separation.md

```
@@ -0,0 +1,180 @@
+## Procedure
+
+### 1. Create the staging branch
+
+```bash
+git checkout main && git pull
+git checkout -b staging
+git push -u origin staging
+```
+
+Or via GitHub MCP:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)